content/help/ops/security/repairing-citadel/index.md

```diff
@@ -22,3 +37,19 @@ debugging information and check if there are any errors:
 $ kubectl logs -l istio=citadel -n istio-system
 $ kubectl describe pod -l istio=citadel -n istio-system
 {{< /text >}}
+
+If you want to check a workload (with `default` service account and `default` namespace)
+certificate's lifetime:
+
+{{< text bash >}}
+$ kubectl get secret -o json istio.default -n default | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -noout -text | grep "Not After" -C 1
+  Not Before: Jun  1 18:23:30 2019 GMT
+  Not After : Aug 30 18:23:30 2019 GMT
+Subject:
+{{< /text >}}
+
+{{< text plain >}}
```

Comment:
can you use {{< tip >}} rather than {{< text >}} and same for closing.

Reply:
Done, and the lint is fixed as well.